api.planx.uk/modules/analytics/metabase/collection/service.ts

```diff
@@ -0,0 +1,45 @@
+import { createMetabaseClient } from "../shared/client.js";
+import { updateMetabaseId } from "./updateMetabaseId.js";
+import type { NewCollectionParams } from "./types.js";
+import { getTeamAndMetabaseId } from "./getTeamAndMetabaseId.js";
+import { createCollection } from "./createCollection.js";
+
+const client = createMetabaseClient();
+
+/**
+ * First uses name to get teams.id and .metabase_id, if present.
+ * If metabase_id is null, return an object that includes its id. If not, create the collection.
+ * @params `name` is required, but `description` and `parent_id` are optional.
+ * @returns `response.data`, so use dot notation to access `id` or `parent_id`.
+ */
+export async function checkCollections(
```

Comment:
Maybe I'd call this `createCollectionIfDoesNotExist` - "checking" is a bit of a vague term

Reply:
Then maybe the controller could just be called `CollectionsController` or even `MetabaseCollectionsController`